package.json

```diff
@@ -10,6 +10,9 @@
         "cy:install": "cd frontend && npm run cy:install",
         "install:all": "npm i && concurrently \"cd shared && npm i\" \"cd frontend && npm i\" \"cd backend && npm i\"",
         "setup": "npm i && cd shared && npm i && npm run build && cd .. && concurrently \"cd frontend && npm i\" \"cd backend && npm i\"",
+        "prune": "npm prune && cd shared && npm prune && cd ../frontend && npm prune && cd ../backend && npm prune",
+        "prune:deployment": "npm prune --production && cd shared && npm prune --production && cd ../backend && npm prune --production",
+        "deployment": "npm run setup:ci && npm run build:deployment && npm run prune:deployment",
```

Comment:
Where are these scripts used?

Reply:
I think you could use `prune:deployment` to make the container even smaller, right?

Reply:
`prune:deployment` is used in `deployment`
having `prune:deployment` without a `prune` would be maybe not great, `prune` can be used locally, when you want to clean up all packages (is not used).

Reply:
Ah, I didn't read the file correctly. All good then here.